### PR TITLE
Changed username and urls to computed to fix the idempotency

### PR DESCRIPTION
**Pull request by anvitha-jain**
solves #224

### CODE REVIEW

File: mso/resource_mso_site.go [hunk 1]
```diff
@@ -79,6 +80,7 @@ func resourceMSOSite() *schema.Resource {
 			"urls": &schema.Schema{
 				Type:     schema.TypeList,
 				Optional: true,
+				Computed: true,
```
Comment (shrsr) [inline mso/resource_mso_site.go:83]: Would it be a good idea to change the "location" attribute to Computed as well?

**Review by shrsr — CHANGES_REQUESTED** [review]
Just the comment above--

File: mso/resource_mso_site.go (hunk 1 above)
Comment (anvitha-jain) [reply mso/resource_mso_site.go:85]: changed.

File: mso/resource_mso_site.go [hunk 2]
```diff
@@ -37,11 +37,13 @@ func resourceMSOSite() *schema.Resource {
 			"username": &schema.Schema{
 				Type:     schema.TypeString,
 				Optional: true,
+				Computed: true,
 			},
 
 			"password": &schema.Schema{
 				Type:     schema.TypeString,
 				Optional: true,
+				Computed: true,
```
Comment (shrsr) [inline mso/resource_mso_site.go:46]: It would also be good to mark the password as Sensitive ->
Sensitive: true

**Review by shrsr — CHANGES_REQUESTED** [review]

**Review by shrsr — DISMISSED** [review]
LGTM

**Review by akinross — APPROVED** [review]
LGTM

**Review by shrsr — CHANGES_REQUESTED** [review]
Adding Computed : True to location is affecting the idempotent behaviour because your Terraform configuration file does not have a value for location. When you try to re-apply after the first apply your state which has now has a value for location is compared with your configuration which finds the location empty.

 Let's discuss with Lionel if we want the omit this.

**Review by shrsr — APPROVED** [review]
LGTM

**Review by akinross — DISMISSED** [review]
LGTM

**Review by lhercot — APPROVED** [review]
LGTM

**Pull request merged**